client/packages/common/src/ui/forms/JsonForms/api/hooks/utils/useDocumentApi.ts

```diff
@@ -0,0 +1,14 @@
+import { useGql, useAuthContext } from '@openmsupply-client/common';
+import { getDocumentQueries } from '../../api';
+import { getSdk } from '../../operations.generated';
+
+export const useDocumentApi = () => {
+  const { storeId } = useAuthContext();
+  const keys = {
+    detail: (id: string) => [storeId, id] as const,
```

Comment:
:nit the usage is fetching by name, so this should be `name: string` rather than `id`

strictly speaking, should be `docName` to match usage, but, as noted, I'd prefer to simplify `docName` anyway